src/main/webapp/app/exercises/text/participate/text-participation.module.ts

```diff
@@ -8,10 +8,10 @@ import { ArtemisSharedComponentModule } from 'app/shared/components/shared-compo
 import { TextEditorComponent } from 'app/exercises/text/participate/text-editor.component';
 import { TextResultComponent } from 'app/exercises/text/participate/text-result/text-result.component';
 import { ArtemisHeaderExercisePageWithDetailsModule } from 'app/exercises/shared/exercise-headers/exercise-headers.module';
-import { ArtemisTeamModule } from 'app/exercises/shared/team/team.module';
 import { ArtemisTeamSubmissionSyncModule } from 'app/exercises/shared/team-submission-sync/team-submission-sync.module';
 import { RatingModule } from 'app/exercises/shared/rating/rating.module';
 import { ArtemisMarkdownModule } from 'app/shared/markdown.module';
+import { ArtemisTeamParticipeModule } from 'app/exercises/shared/team/team-participate/team-participate.module';
 
 const ENTITY_STATES = [...textEditorRoute];
```

Comment:
This could be refactored like in the other places

Reply:
done